Advanced Linux Template : Ansible installation for managing all the VMs.

GONZALORUIZ opened the pull request.

GONZALORUIZ committed:
Ansible installation tested on CentOS 6.5

Initial Commit : Ansible installation tested on CentOS 6.5

GONZALORUIZ committed:
Passwords removed

Passwords removed

singhkays commented:
Hi @GONZALORUIZ 
Few things before I can accept this PR
- can you remove the comments from azuredeploy.json (ARM will run into problems)
- Define metadata description for the parameters in azuredeploy.json
- Create a metadata.json file. 

These steps are outlined here - https://github.com/azurermtemplates/azurermtemplates/blob/master/README.md

GONZALORUIZ committed:
Merge branch 'master' of https://github.com/azurermtemplates/azurermtemplates

GONZALORUIZ committed:
SubscriptionId parameter Removed

SubscriptionId parameter Removed

GONZALORUIZ committed:
Adding GitHub standard files

Comments from the templates removed
Metadata.json file added
Metadata field added to the parameters

GONZALORUIZ committed:
Initial support for Ubuntu

Initial support for Ubuntu

GONZALORUIZ commented:
Thanks Kay, I just incorporated your requests.
